src/functionality.js

```diff
@@ -0,0 +1,100 @@
+const taskContainer = document.querySelector('.task-container');
+const descr = document.querySelector('#addTask');
+
+function Tasks(index, descr, completed) {
+  this.index = index;
+  this.descr = descr;
+  this.completed = completed;
+}
+
+const todotasks = localStorage.getItem('todotasks');
+const taskarr = todotasks ? JSON.parse(todotasks) : [];
+// Adding task
+function addtask() {
+  const description = descr.value;
+  const tasks = new Tasks(taskarr.length, description, false);
+  taskarr.push(tasks);
```

Comment:
- [ ] According to the project requirement, it is expected that the property index is set to the new array length. This means that if you have 5 items in your to-do list, the fifth item's index should be 5.
See the image below to understand better 👇 
![image](https://github.com/manzitresor/To_Do_List/assets/46839548/37cbb0d8-2642-4ebb-afe9-0e50064548ae)